demos/continuous_batching/vlm/README.md

```diff
@@ -0,0 +1,308 @@
+
+import requests
+payload = {"model": "OpenGVLab/InternVL2_5-8B", 
+    "messages": [
+        {
+            "role": "user",
+            "content": [
+              {"type": "text", "text": "Describe what is one the picture."},
+              {"type": "image_url", "image_url": {"url": f"data:image/jpeg;base64,{convert_image('zebra.jpeg')}"}}
+            ]
+        }
+        ],
+    "max_completion_tokens": 100
+}
+headers = {"Content-Type": "application/json", "Authorization": "not used"}
+response = requests.post(base_url + "/chat/completions", json=payload, headers=headers)
+print(response.text)
+```
+```json
+{
+  "choices": [
+    {
+      "finish_reason": "stop",
+      "index": 0,
+      "logprobs": null,
+      "message": {
+        "content": "The picture features a zebra standing in a grassy plain. Zebras are known for their distinctive black and white striped patterns, which help them blend in for camouflage purposes. The zebra pictured is standing on a green field with patches of grass, indicating it may be in its natural habitat. Zebras are typically social animals and are often found in savannahs and grasslands.",
+        "role": "assistant"
+      }
+    }
+  ],
+  "created": 1741731554,
+  "model": "OpenGVLab/InternVL2_5-8B",
+  "object": "chat.completion",
+  "usage": {
+    "prompt_tokens": 19,
+    "completion_tokens": 83,
+    "total_tokens": 102
+  }
+}
+```
+:::
+:::{dropdown} **Streaming request with OpenAI client**
+
+The endpoints `chat/completions` are compatible with OpenAI client so it can be easily used to generate code also in streaming mode:
+
+Install the client library:
+```console
+pip3 install openai
+```
+```python
+from openai import OpenAI
+import base64
+base_url='http://localhost:8080/v3'
+model_name = "OpenGVLab/InternVL2_5-8B"
+
+client = OpenAI(api_key='unused', base_url=base_url)
+
+def convert_image(Image):
+    with open(Image,'rb' ) as file:
+        base64_image = base64.b64encode(file.read()).decode("utf-8")
+    return base64_image
+
+stream = client.chat.completions.create(
+    model=model_name,
+    messages=[
+        {
+            "role": "user",
+            "content": [
+              {"type": "text", "text": "Describe what is one the picture."},
+              {"type": "image_url", "image_url": {"url": f"data:image/jpeg;base64,{convert_image('zebra.jpeg')}"}}
+            ]
+        }
+        ],
+    stream=True,
+)
+for chunk in stream:
+    if chunk.choices[0].delta.content is not None:
+        print(chunk.choices[0].delta.content, end="", flush=True)
+```
+
+Output:
+```
+The picture features a zebra standing in a grassy area. The zebra is characterized by its distinctive black and white striped pattern, which covers its entire body, including its legs, neck, and head. Zebras have small, rounded ears and a long, flowing tail. The background appears to be a natural grassy habitat, typical of a savanna or plain.
+```
+
+:::
+
+## Benchmarking text generation with high concurrency
```

Comment:
With current command we don't really show high concurrency (10 prompts, request rate==1 does not create an impression of high traffic imho)

Reply:
that is about to be updated when concurrency is fixed...